packages/dashboard/src/components/eventDetailsSidebar.tsx

```diff
@@ -0,0 +1,63 @@
+// EventDetailsSidebar.tsx
+import React from "react";
+import { Drawer, Typography, Box } from "@mui/material";
+type SelectedEvent = {
+  [x: string]: any;
+  messageId: string;
+  eventType: string;
+  event: string;
+  userId: string | null;
+  anonymousId: string | null;
+  processingTime: string;
+  eventTime: string;
+  traits: string;
+};
+interface EventDetailsSidebarProps {
+  open: boolean;
+  onClose: () => void;
+  selectedEvent: SelectedEvent | null;
+}
+
+const EventDetailsSidebar: React.FC<EventDetailsSidebarProps> = ({
+  open,
+  onClose,
+  selectedEvent,
+}) => {
+  return (
+    <Drawer
+      open={open}
+      onClose={onClose}
+      anchor="right"
+      slotProps={{ backdrop: { onClick: onClose } }}
+    >
+      <Box padding={2} paddingTop={10}>
+        <Typography fontFamily={"monospace"} variant="h2">
```

Comment:
Can you use the `SubtleHeader` component here?

Reply:
Using SubtleHeader will not allow the font family to be monospace so I used the code instead.

Reply:
In the case of the header, I don't need it to be monospace!